purchasing/templates/opportunities/browse.html

```diff
@@ -14,6 +14,10 @@
       <form class="form form-horizontal" method="POST" action="#">
 
       <div class="spacer-20"></div>
+      <div class="alert alert-warning alert-dismissible" role="alert">
```

Comment:
agree w/@shellicious that this should be in a dropdown/be accessible from the main navbar.